pinot-controller/src/main/java/org/apache/pinot/controller/ControllerStarter.java

```diff
@@ -288,8 +278,10 @@ private void setUpPinotController() {
     // Setting up periodic tasks
     List<PeriodicTask> controllerPeriodicTasks = setupControllerPeriodicTasks();
     LOGGER.info("Init controller periodic tasks scheduler");
-    _controllerPeriodicTaskScheduler = new ControllerPeriodicTaskScheduler();
-    _controllerPeriodicTaskScheduler.init(controllerPeriodicTasks, _controllerLeadershipManager);
+    _periodicTaskScheduler = new PeriodicTaskScheduler();
```

Comment:
let us avoid other gratuitous changes....

Reply:
We don't need all the method in `ControllerPeriodicTaskScheduler`. Thus, we can just remove this class.